.github/workflows/ci_cron_weekly.yml

```
@@ -141,7 +141,8 @@ jobs:
                                   python3-venv \
                                   cython3 \
                                   wcslib-dev \
-                                  liberfa1
+                                  liberfa1 \
+                                  libopenblas-dev
```

```suggestion
                                  libopenblas-dev \
                                  ninja-build
```
This might fix the build error originating from building the `ninja` wheel.